z3c/dependencychecker/modules.py

```diff
@@ -69,12 +69,7 @@ def create_from_files(cls, top_dir):
             return
 
         for path, folders, filenames in os.walk(top_dir):
-            if "__init__.py" not in filenames:
-                # Don't descend further into the tree.
-                # Clear folders variable in-place.
-                folders[:] = []
-                continue
-
+            folders[:] = [d for d in folders if d not in FOLDERS_TO_IGNORE]
```

Comment:
The `__init__.py` rule probably ignored quite some folders. Should we now start to ignore all `.*` folders, too, for instance?

Reply:
On the other hand.... that's something we'll discover in practice and that we can adjust to. I think the implementation is fine.